addons/web/static/src/js/core/dom.js

```diff
@@ -342,15 +343,14 @@ var dom = {
             // a 'real' debounce creation useless. Also, during the debouncing
             // part, the button is disabled without any visual effect.
             $button.addClass('o_debounce_disabled');
+            this.trigger_up('btn_spin_request', {btn: $button[0], spin: true});
+            const always = () => {
+                $button.removeClass('disabled').prop('disabled', false);
```

Comment:
Strange that removing disabled is still needed while adding it is not?

Reply:
Leftover code I didn't really clear the `makeButtonHandler` part as I wasn't sure we really wanted the behavior for it but I just removed it